content/docs/06.workflow-components/11.errors.md

```diff
@@ -3,29 +3,21 @@ title: Errors
 icon: /docs/icons/components.svg
 ---
 
-The `errors` is a list of tasks set on a flow-level that will be executed when an error occurs. You can add multiple such tasks, and they will be executed sequentially.
+There's multiple ways to manage errors in Kestra, to both help you identify them but also allow flows to continue to operate around errors.
+
+## `errors` Component
+
+`errors` is a list of tasks set on a flow-level that will be executed when an error occurs. You can add multiple tasks, and they will be executed sequentially. This is useful for being able to send alerts when errors occur.
 
 The example below sends a flow-level failure alert via Slack using the [SlackIncomingWebhook](/plugins/plugin-notifications/tasks/slack/io.kestra.plugin.notifications.slack.slackincomingwebhook) task defined using the `errors` property.
 
 
-```yaml
-id: unreliable_flow
-namespace: prod
-
-tasks:
-  - id: fail
-    type: io.kestra.plugin.scripts.shell.Commands
-    runner: PROCESS
-    commands:
-      - exit 1
-
-errors:
-  - id: alert_on_failure
-    type: io.kestra.plugin.notifications.slack.SlackIncomingWebhook
-    url: "{{ secret('SLACK_WEBHOOK') }}" # https://hooks.slack.com/services/xyz/xyz/xyz
-    payload: |
-      {
-        "channel": "#alerts",
-        "text": "Failure alert for flow {{ flow.namespace }}.{{ flow.id }} with ID {{ execution.id }}"
-      }
+```yaml file=public/examples/flows_errors_1.yaml
+```
+
+## `allowFailure` Property
+
+When you execute a flow and one of the tasks fail, it will no longer proceed stopping later tasks from executing. This can be annoying if the task that failed doesn't impact other tasks, such as sending a message or a log output. We can resolve this by adding the `allowFailure` property to our task which will let it continue and pass the execution as `WARNING` in the Logs.
```

Comment:
```suggestion
When you execute a flow and one of its tasks fails, downstream tasks won't be executed. This may not always be desirable, especially for non-critical tasks. You can resolve this by adding the `allowFailure` property to the task, which will allow downstream tasks to continue despite the error. In this case, the execution will end in a `WARNING` state.
```